evm-contracts/test/v0.6/ConversionProxy.test.ts

```diff
@@ -0,0 +1,207 @@
+        )
+        matchers.bigNum(
+          ethers.utils.bigNumberify(fiatDecimals),
+          await proxy.decimals(),
+        )
+        assert.equal(newAggregator.address, await proxy.from())
+        assert.equal(newAggregatorFiat.address, await proxy.to())
+      })
+    })
+  })
+
+  describe('#latestAnswer', () => {
+    describe('when converting from ETH to fiat', () => {
+      beforeEach(async () => {
+        await proxy.setAddresses(aggregator.address, aggregatorEth.address)
+        matchers.bigNum(
+          ethers.utils.bigNumberify(asset),
+          await aggregator.latestAnswer(),
+        )
+
+        matchers.bigNum(
+          ethers.utils.bigNumberify(ethAnswer),
+          await aggregatorEth.latestAnswer(),
+        )
+      })
+
+      it('pulls the converted rate from the proxy', async () => {
+        matchers.bigNum(
+          ethers.utils.bigNumberify(convertedEth),
+          await proxy.latestAnswer(),
+        )
+        const latestRound = await proxy.latestRound()
+        matchers.bigNum(
+          ethers.utils.bigNumberify(convertedEth),
+          await proxy.getAnswer(latestRound),
+        )
+      })
+    })
+
+    describe('when converting from fiat to fiat', () => {
+      beforeEach(async () => {
+        matchers.bigNum(
+          ethers.utils.bigNumberify(asset),
+          await aggregator.latestAnswer(),
+        )
+
+        matchers.bigNum(
+          ethers.utils.bigNumberify(fiatAnswer),
+          await aggregatorFiat.latestAnswer(),
+        )
+      })
+
+      it('pulls the converted rate from the proxy', async () => {
+        matchers.bigNum(
+          ethers.utils.bigNumberify(convertedFiat),
+          await proxy.latestAnswer(),
+        )
+        const latestRound = await proxy.latestRound()
+        matchers.bigNum(
+          ethers.utils.bigNumberify(convertedFiat),
+          await proxy.getAnswer(latestRound),
+        )
+      })
+    })
+  })
+
+  describe('#latestTimestamp', () => {
+    beforeEach(async () => {
+      const height = await aggregator.latestTimestamp()
+      assert.notEqual('0', height.toString())
+    })
+
+    it('pulls the timestamp from the proxy', async () => {
+      matchers.bigNum(
+        await aggregator.latestTimestamp(),
+        await proxy.latestTimestamp(),
+      )
+      const latestRound = await proxy.latestRound()
+      matchers.bigNum(
+        await aggregator.latestTimestamp(),
+        await proxy.getTimestamp(latestRound),
+      )
+      matchers.bigNum(
+        await aggregator.getTimestamp(latestRound),
+        await proxy.getTimestamp(latestRound),
+      )
+    })
+  })
+
+  describe('#latestRound', () => {
+    beforeEach(async () => {
+      const height = await aggregator.latestTimestamp()
+      assert.notEqual('0', height.toString())
+    })
+
+    it('pulls the timestamp from the proxy', async () => {
+      matchers.bigNum(await aggregator.latestRound(), await proxy.latestRound())
+    })
+  })
```

Comment:
Can you add a test for `getAnswer` here and make it clear that the intention is it converts historical answers at the current rate? I think that's an important caveat.